CMakeLists.txt

```diff
@@ -180,3 +180,20 @@ if(USE_UPDATER)
   endif()
 
 endif()
+
+# Get the current working branch
+execute_process(
+        COMMAND git rev-parse --abbrev-ref HEAD
```

Comment:
I'd use `git describe` here instead, because it can tell if the working directory code has been modified and does not match what is in the repository - otherwise it is perfectly possible for someone to ~~doctor~~*modify* the source code locally and that not be flagged in the build information as things being different to what is stored.  This is not from any security considerations - the source code is open source - but more so that anyone else looking at the run-time behaviour, say trying to debug a fault - can know that they have the **exact same** code that the other person has.

Reply:
Done.